py/client/pydeephaven/proto/table_pb2_grpc.py

```diff
@@ -602,6 +607,14 @@ def MetaTable(self, request, context):
         context.set_details('Method not implemented!')
         raise NotImplementedError('Method not implemented!')
 
+    def ComputeColumnStatistics(self, request, context):
+        """*
+        Returns a new table representing statistics about a single column of the provided table.
+        """
+        context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+        context.set_details('Method not implemented!')
+        raise NotImplementedError('Method not implemented!')
```

Comment:
Does this need implementation?

Reply:
Yes - every generated implementation looks like this. If you were to implement a python server, you would override this method and offer a real implementation.